pkg/skaffold/config/util.go

```diff
@@ -223,3 +227,34 @@ func IsUpdateCheckEnabled(configfile string) bool {
 	}
 	return cfg == nil || cfg.UpdateCheck == nil || *cfg.UpdateCheck
 }
+
+func ShouldDisplayPrompt(configfile string) bool {
+	if cfg, disabled := isSurveyPromptDisabled(configfile); !disabled {
+		return ifNotSurveyTakenOrPromptNotDisplayed(cfg)
```

Comment:
this can be merged with the `return false` statement. 

```
cfg, disabled := isSurveyPromptDisabled(configfile); 
return !disabled && ifNotSurveyTakenOrPromptNotDisplayed(cfg)
```